client/src/components/applications/profil/DetailPage.js

```diff
@@ -0,0 +1,217 @@
+import React, {Component} from 'react'
+import ReactDOM from 'react-dom'
+import './DetailPage.css'
+
+import * as serverAPI from './actions/serverAPI'
+import Cardboard from './components/Cardboard'
+import DetailKatasterTable from './components/DetailCadastralTable'
+import DetailAssetDeclaration from './components/DetailAssets'
+import MapContainer from './components/MapContainer'
+import {NavLink} from 'react-router-dom'
+import {Row, Col} from 'reactstrap'
+
+class DetailPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      politician: {},
+      kataster: [],
+      katasterAssetsFromDeclaration: [],
+      hnutelnyAssetsFromDeclaration: [],
+      prijmyAssetsFromDeclaration: [],
+      years: [],
+      currentYear: null,
+      reportData: {},
+      mapCenter: {lat: 48.6, lng: 19.5},
```

Comment:
This should be probably moved to constants (e.g. SLOVAKIA = {...}). Btw. I will use this constant too :)